src/views/Events.vue

```diff
@@ -73,56 +75,32 @@ export default {
       }
       return currentEvents;
     },
+
+    //codeIsValid: returns true iff the code is valid.
+    //Needs testing
     codeIsValid() {
-      // In future use regex to prevent leading zeros?
+      //TICKET: Use regex to prevent leading zeros?
       return !isNaN(this.code) && this.code >= 1000 && this.code <= 9999;
     }
   },
+
   methods: {
+    //handleClickInParent: sets the activeEventIndex to the proper eventId
     handleClickInParent(eventId) {
       this.activeEventIndex = eventId - 1;
     },
-    submitCheckin() {
+
+    //submitCheckIn: submits an attempt to check in to an event
+    //Needs testing
+    submitCheckIn() {
       var submittedEvent = this.events[this.activeEventIndex];
       submittedEvent["code"] = this.code;
       submittedEvent["checkin-time"] = Date.now();
+
+      //TICKET: API call needs implementation
```

Comment:
Could you change all occurrences of `TICKET` to `TODO:` with that exact syntax? Then we can use https://marketplace.visualstudio.com/items?itemName=Gruntfuggly.todo-tree to view all of them at once. Sorry that's a lot of busywork changes so no rush on it but will help us keep track of what's in progress. And, if there's a ticket made already we should link to it.